chroma-agent/chroma-agent.spec

```diff
@@ -24,7 +24,8 @@ Requires: python-argparse
 Requires: python-daemon
 Requires: python-setuptools
 Requires: python-requests >= 2.6.0
-Requires: python-tablib yum-utils
+Requires: python2-tablib
```

Comment:
It's part of Red Hat's transition plan to Python 3.  All Python packages going forward have the major version number of the Python they are being built for.  Eventually (i.e. as they are rebuilt) all python-* packages will become python{2,3}-*.